src/components/pages/TodoPage.js

```diff
@@ -8,35 +8,40 @@ class TodoPage extends Component {
 
   render() {
 
+    const { todos } = this.props
```

Comment:
when store data is passed into components, it is accessible through this.props
here I use the  [Object rest spread transform](https://babeljs.io/docs/plugins/transform-object-rest-spread/) to get the branch data from this.props (todos would be in this.props.todos)